### PR TITLE
Php8.2 compatibility

### PR DESCRIPTION
**Pull request by mwijngaard**
Got tests to work again using github actions and added support for php 8.2 using AllowDynamicProperties attribute. Some additional cleanup included.

I didn't upgrade the README yet because I wasn't sure what to write. I guess the whole section on 'updated requirements and features' could be replaced by stating the library is now compatible with composer and PHP version 7.4 -> 8.2.

### CODE REVIEW

**Comment by michieljmmaas** [comment]
Any update on this? 

Would love to see this branch merged!

File: src/Wkt.php
[the remark this answers is not in the document]
Comment (nickolanack) [reply src/Wkt.php:1]: ✅

**Review by nickolanack — APPROVED** [review]

**Comment by michieljmmaas** [comment]
Thanks for merging! 

Could we also get a new release? I would like to upgrade our codebase.

**Comment by mwijngaard** [comment]
@nickolanack mind giving us a release as well?

**Comment by nickolanack** [comment]
Sorry I missed this. You should now have 2.0.13 available to you - Thank you for all your work!

**Pull request merged**